prow/plank/controller.go

```diff
@@ -517,6 +519,7 @@ func (c *Controller) syncTriggeredJob(pj prowapi.ProwJob, pm map[string]coreapi.
 	}
 
 	if pj.Status.State == prowapi.TriggeredState {
+		pj.Status.PendingTime = &pod.CreationTimestamp
```

Comment:
I expected to see unit tests updated for `plank` to reflect this

Reply:
Done.